posthog/api/capture.py

```diff
@@ -89,12 +89,16 @@ def _capture(request, team: Team, event: str, distinct_id: str, properties: Dict
         event=event,
         distinct_id=distinct_id,
         properties=properties,
-        ip=get_ip_address(request),
         team=team,
         **({'timestamp': timestamp} if timestamp else {}),
         **({'elements': elements_list} if elements_list else {})
     )
 
+    ip=get_ip_address(request)
+    if ip:
```

Comment:
Is it possible to put the ip field in the properties during creation? That way we will avoid having to make an extra `UPDATE` query here.

Reply:
Resolved.

Reply:
lgtm! I think just the failing python checks remain.

Reply:
It's safe to ignore those, in fact I'm going to remove that check as the warnings are always low quality.